src/app/pages/LandingPage.svelte

```diff
@@ -1,22 +1,46 @@
 <script lang="ts">
   import Upload from "../components/Upload.svelte";
   import BCP47Tag from "../components//BCP47Tag.svelte";
-  export let name: string;
-
-  let files: any;
 </script>
 
-<style>
-</style>
+<svelte:head>
+  <title> Welcome to Predictive Text Studio </title>
+</svelte:head>
+
+<main>
+  <header>
+    <p role="doc-subtitle">Keyman</p>
+    <h1>Predictive Text Studio</h1>
+
+    <p> Add <strong>prediction</strong> and <strong>autocorrect</strong> to your language </p>
+  </header>
+
+  <section id="explanation">
+    <!-- TODO: figure that explains the flow -->
+
+    <a href="#get-started" class="button button--primary">Get started</a>
+  </section>
+
+  <section id="get-started" class="quick-start">
+    <!-- TODO: should not use hard coded URL! -->
+    <form action="/languages">
+      <fieldset>
+        <legend> Step 1: Select your language </legend>
+        <BCP47Tag />
+      </fieldset>
 
-<main role="application">
-  <h1>Welcome to {name}!</h1>
-  <Upload />
-  <BCP47Tag />
+      <fieldset>
+        <legend> Step 2: Attach a word list </legend>
+        <Upload />
+      </fieldset>
 
-  <p>Hi, my name is Xuechun!</p>
-  <p>Hi my name is Eddie!</p>
-  <p>Pranav</p>
+      <button type="submit"> Upload </button>
+    </form>
+  </section>
 
-  <input type="file" data-testid="input-file" bind:files />
+  <footer>
+    <p><small> © 2020 <a href="https://github.com/eddieantonio/predictive-text-studio/graphs/contributors">
```

Comment:
**[Question]**: I wonder if using decimal code ` &#169;` would make a difference  ?

Reply:
As long as the everything that handles this file corrently deals with UTF-8, there should not be any difference. In the case of U+00A9 COPYRIGHT SIGN, I would prefer the `&copy;` HTML/XML entity, as it's more readable than a random decimal code!